R/transition_risk_profile.R

```diff
@@ -0,0 +1,118 @@
+#' DRAFT of transition risk
+#'
+#' @param products,products_scenarios,scenarios  TODO Data frames.
+#' @param low_threshold,high_threshold  TODO Expressions.
+#'
+#' @return TODO A data frame
+#' @export
+#'
+#' @examples
+#' library(tiltToyData)
+#' library(readr, warn.conflicts = FALSE)
+#' library(dplyr, warn.conflicts = FALSE)
+#'
+#' restore <- options(readr.show_col_types = FALSE)
+#'
+#' products <- read_csv(toy_emissions_profile_products_ecoinvent())
+#' # TODO: Submit a PR to tiltToyData?
+#' products_scenarios <- read_csv(toy_sector_profile_companies()) |>
+#'   select(-c("companies_id", "company_name", "clustered")) |>
+#'   distinct()
+#' scenarios <- read_csv(toy_sector_profile_any_scenarios())
+#'
+#' transition_risk_profile(products, products_scenarios, scenarios)
+#'
+#' options(restore)
+transition_risk_profile <- function(products,
+                                    # TODO: Do we need a new dataset in tiltToyData?
```

Comment:
@maurolepore We can't rename it `transition_risk_profile` because I haven't yet added the code which calculates the risk classification of transition risk scores. This name suits better the upcoming code for transition risk score, hence I request you to consider names like `add_thresholds_transition_risk`/ `thresholds_transition_risk`. Thanks!

Reply:
This PR is just to support thinking and stimulate the conversation. Feel free to take or leave any idea.